pkg/cmd/pulumi/util.go

```diff
@@ -775,13 +775,25 @@ func (cancellationScopeSource) NewScope(events chan<- engine.Event, isPreview bo
 	return c
 }
 
+func makeJSONString(v interface{}) (string, error) {
+	out := bytes.NewBuffer([]byte{})
+	encoder := json.NewEncoder(out)
```

Comment:
The zero value for `bytes.Buffer` is a usable buffer:

```suggestion
	var out bytes.Buffer
	encoder := json.NewEncoder(&out)
```